app/controllers/insured/group_selection_controller.rb

```diff
@@ -184,6 +184,9 @@ def terminate
 
   def edit_plan
     @self_term_or_cancel_form = ::Insured::Forms::SelfTermOrCancelForm.for_view({enrollment_id: params.require(:hbx_enrollment_id), family_id: params.require(:family_id)})
+    return unless @self_term_or_cancel_form.errors.present?
```

Comment:
return will not render from action controller. can you move this to if else condition?